components/brave_wallet_ui/stories/wallet-concept.tsx

```diff
@@ -215,18 +216,18 @@ export const _DesktopWalletConcept = (args: { onboarding: boolean, locked: boole
   const transactions = React.useMemo(() => {
     const response = mockRPCResponse
     const transactionList = response.map((account) => {
-      const id = selectedAsset ? selectedAsset.id : ''
+      const id = selectedAsset ? selectedAsset.symbol : ''
```

Comment:
I think you can do something like `selectedAsset.symbol ?? ''`.

Reply:
Done!